assets/js/app.js

```diff
@@ -1,122 +1,147 @@
-// Executes code when DOM has fully loaded
-$(document).ready(function() {
-
-
-// Create the character object to hold the characters data
 const characters = {
 	"John_Snow": {
 		name: "John Snow",
-		healthPoints: 222, 
-		attack: 10, 
+		healthPoints: 222,
+		attack: 10,
 		image: "assets/images/john_snow.jpg",
 		enemyAttackBack: 15
 	},
 
 	"Arya_Stark": {
 		name: "Arya Stark",
-		healthPoints: 333, 
-		attack: 15, 
+		healthPoints: 333,
+		attack: 15,
 		image: "assets/images/arya_stark.jpg",
 		enemyAttackBack: 10
 	},
 
 	"Grey_Worm": {
 		name: "Grey Worm",
-		healthPoints: 445, 
-		attack: 20, 
+		healthPoints: 445,
+		attack: 20,
 		image: "assets/images/Grey_Worm.jpg",
 		enemyAttackBack: 8
 	},
-	
+
 	"Danny_T": {
 		name: "Daenerys Targaryen",
-		healthPoints: 225, 
-		attack: 28, 
+		healthPoints: 225,
+		attack: 28,
 		image: "assets/images/d_t.jpg",
 		enemyAttackBack: 16
 	}
 };
 
-// Create a function that creates the character cards
-function createCharacterCard (character, cardArea) {
-	// Create a div for each character's card containing name, healthPoints, and image.
-	const $characterDiv = $("<div>", {"class": "characters"});
-	$characterDiv.attr("data-name", character.name);
-	const $characterName = $("<p>");
-	$characterName.attr("class", "characterName");
-	$characterName.text(character.name);
-	const $characterImage = $("<img>");
-	$characterImage.attr("class", "characterImage");
-	$characterImage.attr("src", character.image);
-	const $characterHealthPoints = $("<p>");
-	$characterHealthPoints.attr("class", "characterHealthPoints");
-	$characterHealthPoints.text(character.healthPoints);
-	$characterDiv.append($characterName).append($characterHealthPoints).append($characterImage);
-	$(cardArea).append($characterDiv);
-}
-
-function createCharacter () {
-	for(var key in characters) {
-		console.log(characters[key]);
-		createCharacterCard(characters[key], "#characterArea");
+/**
+ * [description]
+ * @return {[type]} [description]
+ */
+const character = (function(){
+	/**
+	 * [createCharacter description]
+	 * @return {[type]} [description]
+	 */
+	function createCharacter() {
+		for(var key in characters) {
+			console.log(characters[key]);
+			createCharacterCard(characters[key], "#characterArea");
+		}
+	};
+	/**
+	 * [createCharacterCard description]
+	 * @param  {String} character [description]
+	 * @param  {String} cardArea  [description]
+	 * @return {[type]}           [description]
+	 */
+	function createCharacterCard(character, cardArea){
+		// Create a div for each character's card containing name, healthPoints, and image.
```

Comment:
Back in the day jQuery templates would have made this cleaner, but it appears that functionality has been deprecated, however, you may want to take a look at https://www.jsviews.com/ if you think this interface might get more complicated over time.

Reply:
Thanks for the tip!  I can definitely see how this will make the code cleaner.  The double curly braces remind me of React a bit too, making the code more readable.  Can you add classes and data attributes via templating?